polaris-core/src/main/java/org/apache/polaris/core/storage/PolarisStorageConfigurationInfo.java

```diff
@@ -160,7 +161,7 @@ public static Optional<PolarisStorageConfigurationInfo> forEntityPath(
                 Set<String> locations =
                     StorageUtil.getLocationsAllowedToBeAccessed(
```

Comment:
While we're at it, maybe we should rename all the `StorageUtil.getLocationsAllowedToBeAccessed` methods to make it clear that it's not really telling what's "allowed", but really what's *declared* to be used by the table.

Maybe rename it to `StorageUtil.getLocationsDeclaredForTable` or `StorageUtil.getLocationsRequestedForTable` or `StorageUtil.getAllLocationsUsedByTable` or something similar.

Reply:
Fixed